packages/rum-core/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -228,7 +231,8 @@ function newView(
           sessionIsActive,
           eventCounts,
         },
-        viewMetrics
+        viewMetrics,
+        { scrollMetrics }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏add it to the list of properties right above instead of using a separate object